health/target/target_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("Target Constructor", func() {
 		metricIDs = append(metricIDs, mockStr)
 		target, err := target.New(id, true, metricIDs, counterIDs, tCounterIDs)
 		Ω(target).Should(BeNil())
-		Ω(err).ShouldNot(BeNil()) // TODO: check for exact error
+		Ω(err.Error()).Should(Equal("Some metric or counter ID is not unique"))
```

Comment:
please create an error var for this one. You can place it in the top of target.go file
or even better solution. Create health/errors package and place it there together with health/errors.go file